lib/atlas/dataset/derived_dataset.rb

```diff
@@ -4,7 +4,7 @@ class Dataset::DerivedDataset < Dataset
 
     attribute :base_dataset, String
     attribute :scaling,      Preset::Scaling
-    attribute :init,         Hash[Symbol => Float]
+    attribute :init,         Hash[InitializerInput => Float]
```

Comment:
> This is inconsistent with Preset

Yes, but how and when would you decorate the `init` to `InitializerInput` objects? Would that be done in ETEngine?